Packages/PackageSources/meridian-simcache/EFB/SimCache/src/SimCache.tsx

```diff
@@ -0,0 +1,54 @@
+import {
+  App,
+  AppBootMode,
+  AppInstallProps,
+  AppSuspendMode,
+  AppView,
+  AppViewProps,
+  Efb,
+  RequiredProps,
+  TVNode,
+} from "@efb/efb-api";
+import { FSComponent, VNode } from "@microsoft/msfs-sdk";
+
+import "./SimCache.scss";
+
+declare const BASE_URL: string;
+
+class SimCacheView extends AppView<RequiredProps<AppViewProps, "bus">> {
+  public onOpen(): void {}
+
+  public onClose(): void {}
+
+  public onResume(): void {}
+
+  public onPause(): void {}
+
+  public render(): VNode {
+    return <div class="simcache-app">{super.render()}</div>;
```

Comment:
Hmm.  It looks like the definition of `AppView` has this for `render()` -- I'm not entirely sure how to interpret it, but I think it is just related to the `AppContainer`?

```tsx
render() {
  return /* @__PURE__ */ FSComponent.buildComponent(AppContainer, { appViewService: this.appViewService });
}
```

Regardless though, you're right -- we'll probably want to use the app view service since we're planning to utilize the EFB framework pretty soon following this change. :)